app/livechat/server/hooks/markRoomNotResponded.ts

```diff
@@ -15,7 +21,7 @@ callbacks.add(
 		}
 
 		// check if room is yet awaiting for response
-		if (typeof room.t !== 'undefined' && room.t === 'l' && room.waitingResponse) {
+		if (room.t === 'l' && room.isWaitingResponse) {
```

Comment:
I'm not sure about this change here. In some places, I can find `waitingResponse` still being used. 

Update: I think this should be `waitingResponse`. `isWaitingResponse` is being used within apps & its bridges

Reply:
One more thing, we can remove the `room.t === 'l' && ` check here since there's already a similar check b4